tests/cloudpickle_test.py

```diff
@@ -908,6 +908,35 @@ def g():
 
         self.assertEqual(f2.__name__, f.__name__)
 
+    def test_wraps_preserves_function_doc(self):
+        from functools import wraps
+
+        def f():
+            """42"""
+            pass
+
+        @wraps(f)
+        def g():
+            f()
+
+        f2 = pickle_depickle(g)
+
+        self.assertEqual(f2.__doc__, f.__doc__)
+
+    def test_wraps_preserves_function_annotations(self):
+        from functools import wraps
+
+        def f(x: 1) -> float:
```

Comment:
the tests failed in py2 on this syntax. You could add a skipif py2 on this test and set the annotations by manually doing `__annotations__ = {'x': 1}`

Reply:
Better than `eval`, thanks :)